test/e2e/example_cluster_dns.go

```diff
@@ -76,7 +76,7 @@ var _ = Describe("[Example] ClusterDns", func() {
 		namespaces := []*api.Namespace{nil, nil}
 		for i := range namespaces {
 			var err error
-			namespaces[i], err = createTestingNS(fmt.Sprintf("dnsexample%d", i), c)
+			namespaces[i], err = createTestingNS(fmt.Sprintf("dnsexample%d", i), c, nil)
```

Comment:
is there a reason we don't want all of the namespaces created by e2e tests to have the label applied?